draft-ietf-oauth-identity-chaining.md

```diff
@@ -116,26 +120,28 @@ The identity and authorization chaining flow outlined below describes how a comb
 ~~~~
 {: title='Identity and Authorization Chaining Flow'}
 
-The flow illustrated in Figure 1 shows the steps the client in trust Domain A needs to perform to access a protected resource in trust domain B. In this flow, the client has a way to discover the authorization server in Domain B and a trust relationship exists between Domain A and Domain B (e.g., through federation). It includes the following:
+The flow illustrated in Figure 1 shows the steps an actor in Domain A needs to perform to access a protected resource in Domain B. It is assumed, that the actor has a way to discover the authorization server in Domain B and a trust relationship exists between Domain A and Domain B (e.g., through federation). The steps outlined are the following:
 
-* (A) The client of Domain A needs to discover the authorization server of Domain B. See [Authorization Server Discovery](#authorization-server-discovery).
+* (A) The actor needs to discover the authorization server of Domain B. See [Authorization Server Discovery](#authorization-server-discovery).
 
-* (B) The client exchanges its token at the authorization server of its own domain (Domain A) for a JWT authorization grant that can be used at the authorization server in Domain B. See [Token Exchange](#token-exchange).
+* (B) The actor exchanges its token at the authorization server of its own domain (Domain A) for a JWT authorization grant that can be used at the authorization server in Domain B. See [Token Exchange](#token-exchange).
 
-* (C) The authorization server of Domain A processes the request and returns a JWT authorization grant that the client can use with the authorization server of Domain B. This requires a trust relationship between Domain A and Domain B (e.g., through federation).
+* (C) The authorization server of Domain A processes the request and returns a JWT authorization grant that is audienced to the authorization server of Domain B. This requires a trust relationship between Domain A and Domain B (e.g., through federation).
 
-* (D) The client presents the authorization grant to the authorization server of Domain B. See [Access Token Request](#access-token-request).
+* (D) The actor presents the authorization grant to the authorization server of Domain B. See [Access Token Request](#access-token-request).
```

Comment:
I'm not sure this `actor` terminology helps as we are using it to explicitly rename just one component in the flow. Yet the Authorization Server in Domain A also acts in multiple roles but we don't label it an "actor".